WordPress/wordpress-playground
# PHP: Add libjpeg and libwebp support

Adds libjpeg and libwebp support to kitchen-sink and node.js builds.

All PHP versions got libjpeg, but only PHP >= 7.4 got libwebp. Getting it to work on 7.0 – 7.3 will require additional explorations.

Related to https://github.com/WordPress/wordpress-playground/issues/1381 and https://github.com/WordPress/wordpress-playground/issues/1175.

## Testing instructions

* Confirm the CI tests pass
* Go to /phpinfo.php on your local dev server and confirm it lists jpeg and webp support in the "GD" section
